packages/editor-ui/src/Interface.ts

```diff
@@ -117,6 +117,10 @@ export interface INodeTypesMaxCount {
 	};
 }
 
+export interface IExternalHooks {
+	onExternalHookEvent(eventName: string, metadata?: IDataObject): void;
```

Comment:
The name is misleading. It sounds like it would execute something when an external hook event gets called but it does actually the opposite.